File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Microtest/Models/Appointments/Appointment.cs
using System;
using System.Collections.Generic;

namespace NHSOnline.Backend.GpSystems.Suppliers.Microtest.Models.Appointments
{
    public class Appointment
    {
        public string Id { get; set; }

        public string Type { get; set; }

        public string StartTime { get; set; }

        public string EndTime { get; set; }

        public string Location { get; set; }

        public IEnumerable<string> Clinicians { get; set; } = Array.Empty<string>();
        public string Channel { get; set; }
        public string TelephoneNumber { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/microtest/appointments/Appointment.kt
package mocking.microtest.appointments

class Appointment (

    val id: String,
    val type: String,
    val startTime: String,
    val duration: String,
    val endTime: String,
    val location: String,
    val clinicians: List<String>,
    val channel: String,
    val telephoneNumber: String
)
